java/navigate-route/src/main/java/com/esri/arcgisruntime/sample/navigateroute/MainActivity.java

```diff
@@ -17,6 +17,7 @@
 package com.esri.arcgisruntime.sample.navigateroute;
 
 import java.util.ArrayList;
+import java.util.Calendar;
 import java.util.List;
 import java.util.concurrent.ExecutionException;
 
```

Comment:
Are these line breaks intentional?

Reply:
Oh yeah, we used to seperate API imports out. Not possible in Kotlin at the moment, so we're not doing it for kotlin samples.